### PR TITLE
Solution for Text not visible in dark mode #1307

### PR DESCRIPTION
**Pull request by nestorlazcano-fintecheando**
## Description
Solution for Text not visible in dark mode #1307 to correctly display text in tables and alerts

## Related issues and discussion
#1307

## Screenshots, if any

image

image

### CODE REVIEW

**Comment by bharathcgowda** [comment]
Looks ok to me, @BLasan  could you help us by reviewing the code?
@nestorlazcano  can you review why the checks have failed?

File: src/app/app.module.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ import { AppRoutingModule } from './app-routing.module';
     HammerModule,
     HttpClientModule,
     ServiceWorkerModule.register('./ngsw-worker.js', { enabled: environment.production }),
-    TranslateModule.forRoot(),
+    //TranslateModule.forRoot(),
```
Comment (BLasan) [inline src/app/app.module.ts:58]: Remove commented codes

File: src/app/core/http/api-prefix.interceptor.ts [hunk 2]
```diff
@@ -18,7 +18,14 @@ export class ApiPrefixInterceptor implements HttpInterceptor {
    * Intercepts a Http request and prefixes it with `environment.serverUrl`.
    */
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    request = request.clone({ url: environment.serverUrl + request.url });
+
+    /**
+     * Ignore URLs that are complete for i18n
+     */
+    if (!request.url.includes('http:') && !request.url.includes('https:')) {
```
Comment (BLasan) [inline src/app/core/http/api-prefix.interceptor.ts:25]: Would you mind elaborating the logic of this if condition?

File: src/theme/pink-bluegrey.scss [hunk 3]
```diff
@@ -1 +1,9 @@
 @import "~@angular/material/prebuilt-themes/pink-bluegrey.css";
+
+div.alert {
+    background-color: dimgray !important;
+}
+
+table.mat-table tbody tr:nth-child(even) {
+    background-color: dimgray !important;
+}
```
Comment (BLasan) [inline src/theme/pink-bluegrey.scss:9]: Keep an empty line at the EOF

File: src/theme/purple-green.scss [hunk 4]
```diff
@@ -1 +1,9 @@
 @import "~@angular/material/prebuilt-themes/purple-green.css";
+
+div.alert {
+    background-color: dimgray !important;
+}
+
+table.mat-table tbody tr:nth-child(even) {
+    background-color: dimgray !important;
+}
```
Comment (BLasan) [inline src/theme/purple-green.scss:9]: Same as the above

**Review by BLasan — CHANGES_REQUESTED** [review]

File: src/app/app.module.ts (hunk 1 above)
Comment (nestorlazcano-fintecheando) [reply src/app/app.module.ts]: I already removed the commented line

File: src/app/core/http/api-prefix.interceptor.ts (hunk 2 above)
Comment (nestorlazcano-fintecheando) [reply src/app/core/http/api-prefix.interceptor.ts:25]: This solution is part of another PR that I also sent to correct the i18n

File: src/theme/pink-bluegrey.scss (hunk 3 above)
Comment (nestorlazcano-fintecheando) [reply src/theme/pink-bluegrey.scss]: resolved

File: src/theme/purple-green.scss (hunk 4 above)
Comment (nestorlazcano-fintecheando) [reply src/theme/purple-green.scss]: resolved

**Comment by Jov03** [comment]
@nestorlazcano-fintecheando this PR contains commits to fix #1304 also, so please update your PR so that it contains commits to fix this issue only.

**Comment by nestorlazcano-fintecheando** [comment]
Thank you, I will send my new PR

**Comment by nestorlazcano-fintecheando** [comment]
## I already sent the new PR 
https://github.com/openMF/web-app/pull/1354

**Pull request closed without merging**